test/lit/wasm-split/mismatched-hashes.wast

```diff
@@ -15,6 +15,9 @@
 ;; CHECK-SAME: The split module must be the original module that was instrumented
 ;; CHECK-SAME: to generate the profile.
 
+;; Check that the matcing module succeeds
```

Comment:
matching* (But given that it is already merged, maybe let's fix this in the next PR, or nevermind...)

Reply:
Oops, thanks for the catch. I'll keep it in mind to fix in the next PR.